doc/man3/SSL_CTX_set_cert_verify_callback.pod

```diff
@@ -38,14 +38,15 @@ All values, including 0, are ignored
 if the verification mode is B<SSL_VERIFY_NONE>.
 Otherwise, when the return value is 0, the handshake will fail.
```

Comment:
when the return value is less than or equal to 0?

Reply:
OK, done